DQM/TrackingMonitor/src/TrackingMonitor.cc

```diff
@@ -500,17 +500,17 @@ void TrackingMonitor::bookHistograms(DQMStore::IBooker & ibooker,
        NumberOfRecHitsPerTrackVsLUMI->setAxisTitle("scal lumi [10e30 Hz cm^{-2}]",1);
        NumberOfRecHitsPerTrackVsLUMI->setAxisTitle("Mean number of vertices",2);
 
-       double GoodPVtxMin   = conf->getParameter<double>("GoodPVtxMin");
-       double GoodPVtxMax   = conf->getParameter<double>("GoodPVtxMax");
+       double PVMin   = conf->getParameter<double>("PVMin");
+       double PVMax   = conf->getParameter<double>("PVMax");
        
        histname = "NumberOfGoodPVtxVsLUMI";
-       NumberOfGoodPVtxVsLUMI = ibooker.bookProfile(histname,histname,LUMIBin,LUMIMin,LUMIMax,GoodPVtxMin,3.*GoodPVtxMax,"");
+       NumberOfGoodPVtxVsLUMI = ibooker.bookProfile(histname,histname,LUMIBin,LUMIMin,LUMIMax,PVMin,3.*PVMax,"");
        NumberOfGoodPVtxVsLUMI->getTH1()->SetCanExtend(TH1::kAllAxes);
```

Comment:
Hi, 
this is not touched by this PR, but can these be removed as well (in this or a separate pull request)?